tests/Integrations/PCNTL/PCNTLTest.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+namespace DDTrace\Tests\Integrations\PCNTL;
+
+use DDTrace\Tests\Common\IntegrationTestCase;
+use DDTrace\Tests\Common\SpanAssertion;
+
+const ACCEPTABLE_TEST_EXECTION_TIME_S = 2;
+
+final class PCNTLTest extends IntegrationTestCase
+{
+    protected function ddSetUp()
+    {
+        $this->resetRequestDumper();
+        parent::ddSetUp();
+    }
+
+    /**
+     * @dataProvider dataProviderAllScripts
+     */
+    public function testDoesNoHangAtShutdownWhenDisabled($scriptPath)
+    {
+        if ($scriptPath === (__DIR__ . '/scripts/long-running-manual-flush.php')) {
+            $this->markTestSkipped('manual tracing cannot be done when disabled as classes are not available.');
```

Comment:
Tiny nit: it look me a few minutes to figure out what this message was saying.

```suggestion
            $this->markTestSkipped('manual tracing cannot be done when the tracer is disabled because the "DDTrace\\*" classes are not available.');
```

Reply:
😄  thanks